platform/ios/src/MGLMapView.mm

```diff
@@ -1016,6 +1016,8 @@ - (void)glkView:(__unused GLKView *)view drawInRect:(__unused CGRect)rect
 - (void)layoutSubviews
 {
     [super layoutSubviews];
+    
+    [self.scaleBar invalidateIntrinsicContentSize];
```

Comment:
Can we move this line to before `[super layoutSubviews]` so that the size is correct in the current layout call?